clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -191,15 +190,16 @@ func (w *waitForControlledPodsRunningMeasurement) gather(syncTimeout time.Durati
 	var numberRunning, numberDeleted, numberTimeout, numberUnknown int
 	unknowStatusErrList := errors.NewErrorList()
 	timedOutObjects := []string{}
-	for _, checker := range w.checkerMap {
-		status, err := checker.getStatus()
+	for _, checker := range (map[string]checker.Checker)(w.checkerMap) {
+		objChecker, _ := (checker).(*objectChecker)
```

Comment:
Don't silently ignore error.
If you're sure that the checker is of the type just do:

objCheck := checker.(*objectChecker)

Reply:
Done.